src/app/History.ts

```diff
@@ -1,10 +1,15 @@
-import { createBrowserHistory, createMemoryHistory } from 'history';
+import { createBrowserHistory, createMemoryHistory, createHashHistory } from 'history';
 import { toValidDuration } from '../config/ServerConfig';
 import { BoundsInMilliseconds } from 'types/Common';
 
 const webRoot = (window as any).WEB_ROOT ? (window as any).WEB_ROOT : undefined;
 const baseName = webRoot && webRoot !== '/' ? webRoot + '/console' : '/console';
-const history = process.env.TEST_RUNNER ? createMemoryHistory() : createBrowserHistory({ basename: baseName });
+const historyMode = (window as any).HISTORY_MODE ? (window as any).HISTORY_MODE : 'browser';
+const history = process.env.TEST_RUNNER
+  ? createMemoryHistory()
+  : historyMode === 'browser'
```

Comment:
I think this 'kind-of' is making `createHashHistory` to be the "standard" default, because an arbitrary string in the `web_history_mode` will have the effect to using hash-history.

Since `createBrowserHistory` is what Kiali has been using for a long time, I prefer to change this conditional so that Kiali will fallback to `createBrowserHistory` if the value of the history mode is unknown.

Reply:
```js
const historyMode = (window as any).HISTORY_MODE ? (window as any).HISTORY_MODE : 'browser';
const history = process.env.TEST_RUNNER
  ? createMemoryHistory()
  : historyMode === 'hash'
  ? createHashHistory()
  : createBrowserHistory({ basename: baseName });
```
Sorry, I haven't paid attention for too long

Is this method feasible?